src/lib/proxy-server.ts

```diff
@@ -81,6 +82,33 @@ export async function startProxyServer(): Promise< boolean > {
 			} );
 		} );
 
+		// On Windows, we need to check if the port is in use before we can listen on it
```

Comment:
```suggestion
		// On Windows, node doesn't throw an error if port 80 is busy, so we use the net module to explicitly check
		// if it's possible to establish a TCP connection to that port (meaning it's busy)
```
I think it's worth clarifying why we do this